packages/manager/src/queries/databases.ts

```diff
@@ -182,6 +182,7 @@ export const databaseEventsHandler = (event: Event) => {
           // Database status will change from `provisioning` to `active` (or `failed`) and
           // the host fields will populate. We need to refetch to get the hostnames.
           queryClient.invalidateQueries([queryKey, entity!.id]);
+          queryClient.invalidateQueries(`${queryKey}-list`);
```

Comment:
I'm still trying to figure out React Query, so let me know if this is bad or unnecessary!

I don't think this is strictly necessary for the E2E tests, but I noticed that when getting the `database_create` `finished` event, the landing page statuses did not immediately refresh and instead took a few seconds to reflect the changes (I assume this corresponds to the `refetchInterval` specified in line 51 of this file). This delay occasionally causes the test to time out and I assumed this was not the intended behavior.

Reply:
I'm good with that change!

Alternatively we could say this below to avoid hitting the API again, but refetching is nice to ensure we have accurate data. Since the event returns a status, I think this approach is pretty safe. You can make the final decision 🙂

```suggestion
          updatePaginatedDatabaseStore(entity!.id, {
            status: status === 'failed' ? 'failed' : 'active',
          });
```